.github/actions/godot-deps/action.yml

```diff
@@ -29,3 +29,7 @@ runs:
         python -c "import sys; print(sys.version)"
         python -m pip install scons==${{ inputs.scons-version }}
         scons --version
+
+    - name: Setup problem matchers
+      shell: bash
+      run: echo ::add-matcher::misc/utility/problem-matchers.json
```

Comment:
This is a magic command for GitHub?

Reply:
[Yup](https://github.com/actions/toolkit/blob/main/docs/commands.md#problem-matchers)! Didn't know it even existed until today